bundles/org.openhab.binding.haassohnpelletstove/pom.xml

```
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>3.1.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.binding.haassohnpellestove</artifactId>
```

That's the reason why you can't compile it:
```suggestion
  <artifactId>org.openhab.binding.haassohnpelletstove</artifactId>
```

---

wow, that made me nuts. Thanks @fwolter you saved my day. Will do the rest of the suggested changes. 
It does compile again.

---

There are even more typos with it in other files I had to fix. thanks for help. It runs again.